portals/bring-your-own-code-samples/react-sample/README.md

```diff
@@ -0,0 +1,123 @@
+# Power Pages - Bring Your Own Code - React Sample
+
+This is a react based sample application to demonstrate how a react website can be hosted on Power Pages and integrate with features like authentication, authorization, web apis etc.
+
+This is a modern web application for managing car sales, inventory, and customer information.
+
+## Features
+
+- Authentication using Microsoft Entra Id (see [AuthButton.tsx](src/components/AuthButton.tsx) for implementation)
+- Authorization using Power Pages web roles (see [Layout.tsx:45-49](src/components/Layout.tsx#L45-L49) for implementation)
+- Fetching data from external data sources through [virtual tables](https://learn.microsoft.com/en-us/power-pages/configure/virtual-tables) and web apis (see [SalesLeads.tsx](src/pages/SalesLeads.tsx#L30-L63) for implementation)
+
+## Prerequisites
+
+- Node.js (v14 or higher)
+- npm (v6 or higher)
+
+## Installation
+
+1. Clone the repository:
+
+    ```bash
```

Comment:
Why `bash`?  For Windows, you need to enable it.
We typically use `powershell` since people testing this will probably use windows.

Reply:
This was just for syntax highlighting. I have updated to powershell.